### PR TITLE
DEVDOCS-5957 [new]: MSF International Enhancements, add custom_fields

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5957]

## What changed?
- (GQL Admin API) Update MSF International Enhancements overview page to mention custom fields
- (GQL Admin API) Add new custom fields doc for example requests/responses
- (Webhooks) Add new custom field webhook
- (GQL Storefront API) Mention that API responses depend on channel locale (& control panel settings)
- (Stencil) Mention that Stencil storefront channels support overrides for localization 

## Release notes draft

* 

## Anything else?

Related PRs:
- [PR 3090](https://github.com/bigcommerce/storefront/pull/3090) (GQL SF API descriptions)
- [PR 2975](https://github.com/bigcommerce/api-proxy-java/pull/2975) (GQL Admin API descriptions)
- [PR 941](https://github.com/bigcommerce/developer-center/pull/941) (TOC changes for dev center)

ping {names}

[DEVDOCS-5957]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5957?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/store-operations/catalog/graphql-admin/product-basic-info.mdx
Comment (bc-andreadao) [inline docs/store-operations/catalog/graphql-admin/product-basic-info.mdx:36]: After this PR is merged, I'm planning on moving this table to the main overview page (instead of having it on the Product Basic Information page)

File: docs/integrations/webhooks/events/index.mdx
Comment (kristinapototska) [inline docs/integrations/webhooks/events/index.mdx:862]: @bc-andreadao this ticket has a description of how custom fields webhook works https://bigcommercecloud.atlassian.net/browse/CATALOG-9917 

It was added as one of the `properties` to `store/product/updated` webhook. There's no separate webhook for custom fields. Can you please share which ticket you were using for documentation?

**Review by kristinapototska — CHANGES_REQUESTED** [review]

File: docs/integrations/webhooks/events/index.mdx
Comment (bc-andreadao) [reply docs/integrations/webhooks/events/index.mdx:862]: @kristinapototska I was using https://bigcommercecloud.atlassian.net/browse/MSF-3602 which talks about `store/product/customFields/updated`. The ticket was also linked to https://bigcommercecloud.atlassian.net/browse/PROJECT-3973

I can take it out though and only document CATALOG-9917

Comment (jyuricle) [reply docs/integrations/webhooks/events/index.mdx:862]: @bc-andreadao you can find the example of payload for webhook triggerred after custom fields updates on the screenshot in https://github.com/bigcommerce/webhooks-manager/pull/1173. This is the only supported case for custom fields updates right now

Comment (bc-andreadao) [reply docs/integrations/webhooks/events/index.mdx:862]: @jyuricle Thanks, that URL is giving me a 404 error. It would be great if you could send me the payload screenshot or give me access to the repository

File: docs/store-operations/catalog/graphql-admin/product-custom-fields.mdx
Comment (jyuricle) [inline docs/store-operations/catalog/graphql-admin/product-custom-fields.mdx:123]: I guess here the structure should contain connection nodes (`"edges"` and `"node"`) as well, the similar as in the example below

File: docs/integrations/webhooks/events/index.mdx
Comment (jyuricle) [reply docs/integrations/webhooks/events/index.mdx:862]: Screenshot 2024-07-08 at 18 44 08

File: docs/store-operations/catalog/graphql-admin/product-custom-fields.mdx
Comment (bc-andreadao) [reply docs/store-operations/catalog/graphql-admin/product-custom-fields.mdx]: Thanks for catching, I just tested / changed the examples

File: docs/integrations/webhooks/events/index.mdx
Comment (bc-andreadao) [reply docs/integrations/webhooks/events/index.mdx]: Thanks, it's fixed now

File: docs/store-operations/catalog/graphql-admin/product-basic-info.mdx
Comment (kristinapototska) [inline docs/store-operations/catalog/graphql-admin/product-basic-info.mdx:36]: @bc-andreadao it might be worth to include links to the API to query the data in a context of a channel and locale here as well, for the events when webhook payload has a context in it. We can address this latter as a separate ticket and add this for product, options, modifiers webhooks.

**Review by kristinapototska — APPROVED** [review]
Looks good to me :) 
But I'd love to get either @jyuricle or @andrii-lenets approval once they review custom fields GQL API schema to make sure the documentation matches the implementation. And we are good to go once any of them approve

**Review by jyuricle — APPROVED** [review]
LGTM

**Pull request merged**
